daisy_workflows/styleguide.md

```diff
@@ -0,0 +1,61 @@
+# Daisy Workflow Styleguide
+Recommendations from the GCE team about how to style daisy workflows.
+
+## Workflow file naming
+Use `.wf.json` for workflow names and prefer underscores in file names.
+
+For example: `my_awesome_build.wf.json`.
+
+## Workflow naming
+Use dashed names for workflow names. `Name": "my-awesome-build"`
+
+## Step naming
+Don't use camel case for step naming as this looks too much like the Steps
+themselves. Instead, use dashed names: `create-build-disk` instead of
+`createBuildDisk` or `CreateBuildDisk`.
+
+For example:
+
+```json
+"Steps": {
+  "create-build-disk": {
+    "CreateDisks": [
+      {
+        "Name": "disk-debian-build",
+        "SourceImage": "projects/bct-prod-images/global/images/family/debian-8",
+        "SizeGb": "10",
+        "Type": "pd-ssd"
+      }
+    ]
+  }
+}
+```
+
+## Resource naming
+Name resources with a prefix denoting the resource type. This will make it much
```

Comment:
I can see what you're suggesting here, but this makes resource names longer -- not the end of the world, but undesirable.

FYI, semi-relevant side note: it is OK to have a disk "foo" AND an instance "foo" simultaneously. There isn't a naming collision.